src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,74 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileReader;
+import java.io.BufferedReader;
+
+/**
+ * Class for storing and loading tasks from a file.
+ */
+public class Storage {
+    /**
+     * Saves the contents of a TaskList into the a file.
+     * @param list - the list of tasks to load into
+     */
+    public static void save(TaskList list) {
+        try {
+            File file = new File("./data/duke.txt");
+            if (!file.getParentFile().exists()) {
+                file.mkdirs();
+            }
+            file.createNewFile();
+            FileWriter writer = new FileWriter("./data/duke.txt", false);
+            writer.write(list.saveData());
+            writer.close();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Loads the contents of a TaskList into the program.
+     * @param filePath - the name of the file to load from
+     * @return the list of tasks loaded from the file
+     */

```

Comment:
For all JavaDoc comments, take note to leave an empty line between the description and parameter section and put punctuation behind each parameter description. Maybe the dash behind @param can be omitted as well just to follow the coding standards.